test/gtest/uct/v2/test_uct_query.cc

```diff
@@ -69,3 +104,76 @@ UCS_TEST_P(test_uct_query, query_perf)
 }
 
 UCT_INSTANTIATE_TEST_CASE(test_uct_query)
+
+class test_uct_query_ib : public test_uct_query {
+public:
+    double get_attr_latency_c() const;
+};
+
+double test_uct_query_ib::get_attr_latency_c() const
+{
+    return get_iface_attr().latency.c;
+}
+
+UCS_TEST_P(test_uct_query_ib, send_overhead,
+           "IB_SEND_OVERHEAD=bcopy:" UCS_PP_MAKE_STRING(IB_SEND_OVERHEAD_BCOPY)
+           ",cqe:" UCS_PP_MAKE_STRING(IB_SEND_OVERHEAD_CQE) ",db:"
+           UCS_PP_MAKE_STRING(IB_SEND_OVERHEAD_DB) ",wqe_fetch:"
+           UCS_PP_MAKE_STRING(IB_SEND_OVERHEAD_WQE_FETCH) ",wqe_post:"
+           UCS_PP_MAKE_STRING(IB_SEND_OVERHEAD_WQE_POST))
```

Comment:
Minor: IMO using `UCS_PP_MAKE_STRING` is redundant here, without it code is shorter and easier to read: 
```suggestion
           "IB_SEND_OVERHEAD=bcopy:" #IB_SEND_OVERHEAD_BCOPY
           ",cqe:" #IB_SEND_OVERHEAD_CQE
           ",db:" #IB_SEND_OVERHEAD_DB
           ",wqe_fetch:" #IB_SEND_OVERHEAD_WQE_FETCH
           ",wqe_post:" #IB_SEND_OVERHEAD_WQE_POST)
```

Reply:
The only way to convert a macro argument into a character constant is to use two levels of macros.
```C
#define _UCS_PP_MAKE_STRING(x) #x
#define  UCS_PP_MAKE_STRING(x) _UCS_PP_MAKE_STRING(x)
...
#define IB_SEND_OVERHEAD_BCOPY 1
...
/**
  * #IB_SEND_OVERHEAD_BCOPY                     -> "IB_SEND_OVERHEAD_BCOPY"
  * _UCS_PP_MAKE_STRING(IB_SEND_OVERHEAD_BCOPY) -> "IB_SEND_OVERHEAD_BCOPY"
  *  UCS_PP_MAKE_STRING(IB_SEND_OVERHEAD_BCOPY) -> "1"
  */
```